### ISSUE TITLE
Pretrained model of AudiovisualEmotionLearner

### ISSUE THREAD

**Issue by charsyme** [opendr-eu/opendr#505]
Which actors from the RAVDESS dataset were utilized for training the provided model of the AudiovisualEmotionLearner? 
Based solely on the documentation, the provided information is as follows:

"This method is used to download a pretrained model from a given directory. Pretrained models are provided for the 'ia' fusion with 'zerodrop' modality dropout. The pretrained model is trained on [RAVDESS](https://zenodo.org/record/1188976#.YlkXNyjP1PY) dataset which is under CC BY-NC-SA 4.0 license."
